eth/stagedsync/stage_polygon_sync.go

```diff
@@ -0,0 +1,13 @@
+package stagedsync
+
+func SpawnPolygonSyncStage() error {
+	return nil
+}
+
+func UnwindPolygonSyncStage() error {
+	return nil
```

Comment:
hmm?

Reply:
as per the description in the PR: "Subsequent PRs will build on top of this and add the necessary implementation logic." - this is just a skeleton to plug in the new stage loop